app/code/Magento/SalesRule/Model/ResourceModel/Coupon/Usage.php

```diff
@@ -25,11 +25,13 @@ protected function _construct()
     /**
      * Increment times_used counter
      *
-     * @param int $customerId
-     * @param mixed $couponId
-     * @return void
+     * @param  int $customerId
+     * @param  mixed $couponId
+     * @param int|null $delta
+     *
+     * @throws \Magento\Framework\Exception\LocalizedException
      */
-    public function updateCustomerCouponTimesUsed($customerId, $couponId)
+    public function updateCustomerCouponTimesUsed($customerId, $couponId, $delta = 1)
```

Comment:
Unfortunately, we can't change the public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
Original method updateCustomerCouponTimesUsed restored
Custom method added instead
Please check.